api/v1beta1/ssp_webhook_test.go

```diff
@@ -119,7 +119,7 @@ var _ = Describe("SSP Validation", func() {
 		})
 	})
 
-	It("should not allow update of commonTemplates.namespace", func() {
+	It("should allow update of commonTemplates.namespace", func() {
```

Comment:
You can remove this test. The webhook does not restrict the namespace anymore.

Reply:
I would prefer to keep it there, because it should check that webhook doesn't prevent to change namespace